frontend/packages/dev-console/src/components/import/git/__tests__/GitSection.spec.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { render, waitFor, screen, configure, cleanup, fireEvent } from '@testing-library/react';
+import { Formik, FormikConfig } from 'formik';
+import { act } from 'react-dom/test-utils';
+import { Provider } from 'react-redux';
+import { GitProvider } from '@console/git-service/src';
+import store from '@console/internal/redux';
+import { devStrategyInput } from '../../../../../integration-tests/testData/git-import-flow.data';
```

Comment:
When just used in this test case, you can also inline the mock data or create a DevfileStrategySection.data.tsx parallel to this file. These cross imports from a unit test to our e2e make it difficult to maintain this later.